src/lib/rmt/maria_db/current_root_password_dialog.rb

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2018 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+require 'rmt/base'
+
+module RMT; end
+module RMT::MariaDB; end
+
+class RMT::MariaDB::CurrentRootPasswordDialog < RMT::Base
+  def run
+    ret = nil
+
+    UI.OpenDialog(
+      VBox(
+        VSpacing(1),
+        Heading(_('Database root password is required')),
```

Comment:
when translation is used, then file has to have textdomain call otherwise pot file extraction failed. I found we miss it in our documentation, but rule is any file that contain translatable string has to have one textdomain call. Example is e.g. https://github.com/yast/yast-bootloader/blob/master/src/lib/bootloader/config_dialog.rb#L41

it is needed to associate strings with textdomain.

Reply:
I see that `check:pot` task is failing now :grimacing::+1: